code/src/App.js

```diff
@@ -1,9 +1,74 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import ThoughtForm from './components/Thought-Form'
```

Comment:
A small detail I found from the common pitfalls for week 9 is that outer dependencies should be divided from local by one empty line. And I'm only mentioning this because I think your coding overall is good and small details like this can only make the code look even better :D